src/resmom/mom_hook_func.c

```diff
@@ -3075,6 +3077,285 @@ record_job_last_hook_executed(unsigned int hook_event,
 	fclose(fp);
 }
 
+ static int run_execjob_end_hooks(struct work_task *);
+
+/**
+ * @brief
+ * This function runs after execution of a single execjob_end hook and
+ * process the results of hook, on successful hook execution new task
+ * will be created to run the next hook script. If there was  an
+ * error (the hook process returned a non-zero exit status) it does
+ * not create the new task for the next hook script and sends replies
+ * to the outstanding batch request.
+ * 
+ * @param[in] 	ptask - the work task.
+ *
+ * @return none
+ */
+static void
+post_execjob_end_hook(struct work_task *ptask)
+{
+
+	int    accept_flag = 1;
+	int    reject_flag = 0;
+	int    reject_rerunjob = 0;
+	int    reject_deletejob = 0;
+	int    reboot_flag = 0;
+	int    log_type = 0;
+	int    log_class = 0;
+	int    hook_error_flag = 0;
+	int    wstat = ptask->wt_aux;
+	char   *log_id = NULL;
+	char   reject_msg[HOOK_MSG_SIZE+1] = {'\0'};
+	char   reboot_cmd[HOOK_BUF_SIZE]  = {'\0'};
+	char   hook_outfile[MAXPATHLEN+1] = {'\0'};
+	pid_t  mypid = ptask->wt_event;
+	hook   *phook	= (hook *)ptask->wt_parm1;
+	mom_hook_input_t *hook_input 	= (mom_hook_input_t *) ptask->wt_parm2;
```

Comment:
same here, check if ptask is not NULL first.

Reply:
Done